integration-tests/universal/log_poller/helpers.go

```diff
@@ -81,71 +80,72 @@ var registerSingleTopicFilter = func(registry contracts.KeeperRegistry, upkeepID
 }
 
 // this is not really possible, log trigger doesn't support multiple topics, even if log poller does
-var registerMultipleTopicsFilter = func(registry contracts.KeeperRegistry, upkeepID *big.Int, emitterAddress common.Address, topics []abi.Event) error {
-	if len(topics) > 4 {
-		return errors.New("Cannot register more than 4 topics")
-	}
-
-	var getTopic = func(topics []abi.Event, i int) common.Hash {
-		if i > len(topics)-1 {
-			return bytes0
-		}
-
-		return topics[i].ID
-	}
-
-	var getFilterSelector = func(topics []abi.Event) (uint8, error) {
-		switch len(topics) {
-		case 0:
-			return 0, errors.New("Cannot register filter with 0 topics")
-		case 1:
-			return 0, nil
-		case 2:
-			return 1, nil
-		case 3:
-			return 3, nil
-		case 4:
-			return 7, nil
-		default:
-			return 0, errors.New("Cannot register filter with more than 4 topics")
-		}
-	}
-
-	filterSelector, err := getFilterSelector(topics)
-	if err != nil {
-		return err
-	}
-
-	logTriggerConfigStruct := automation_utils_2_1.LogTriggerConfig{
-		ContractAddress: emitterAddress,
-		FilterSelector:  filterSelector,
-		Topic0:          getTopic(topics, 0),
-		Topic1:          getTopic(topics, 1),
-		Topic2:          getTopic(topics, 2),
-		Topic3:          getTopic(topics, 3),
-	}
-	encodedLogTriggerConfig, err := automationUtilsABI.Methods["_logTriggerConfig"].Inputs.Pack(&logTriggerConfigStruct)
-	if err != nil {
-		return err
-	}
-
-	err = registry.SetUpkeepTriggerConfig(upkeepID, encodedLogTriggerConfig)
-	if err != nil {
-		return err
-	}
-
-	return nil
-}
-
-func NewOrm(logger core_logger.SugaredLogger, chainID *big.Int, postgresDb *ctf_test_env.PostgresDb) (*lpEvm.DbORM, *sqlx.DB, error) {
+// Unused???
```

Comment:
Are these planned to be used @Tofel

Reply:
Yes, it's not used currently. I just left it here for the future, since I have written it before I knew that log trigger doesn't support multiple triggers.

If we don't want dead code please feel free to remove it.

Reply:
I will put a note with a date on it in case you need it in the future.